src/cs-core/GuiManager.hpp

```diff
@@ -123,6 +123,12 @@ class CS_CORE_EXPORT GuiManager {
   /// @param jsFile The javascript file that contains the source code.
   void addScriptToSideBarFromJS(std::string const& jsFile);
```

Comment:
These two methods should be called `addScriptToGui` and `addScriptToGuiFromJS`.

Reply:
Renamed.